tests/shell/shell_recorder/shell_recorder.sh

```diff
@@ -208,54 +208,54 @@ execute()
 
 run_script()
 {
-    while read line;
+    while read -r line;
     do
         OP=
         ARG=
-        cmd=$(echo $line|cut -d ' ' -f1)
+        cmd=$(echo "$line"|cut -d ' ' -f1)
         case "$cmd" in
             Mountpoint:)
-                Mountpoint=$(echo $line|cut -d ' ' -f2)
+                Mountpoint=$(echo "$line"|cut -d ' ' -f2)
                 ;;
             File:)
-                DBFile=$(echo $line|cut -d ' ' -f2)
+                DBFile=$(echo "$line"|cut -d ' ' -f2)
                 if [ "$DBFile" = "File:" -o -z "$DBFile" ]; then
 			DBFile=$(mktemp -t elektraenv.XXXXXXXXX 2>/dev/null || mktemp -t 'elektraenv')
                 fi
                 ;;
             Storage:)
-                Storage=$(echo $line|cut -d ' ' -f2)
+                Storage=$(echo "$line"|cut -d ' ' -f2)
                 ;;
             MountArgs:)
-                MountArgs=$(echo $line|cut -d ' ' -f2-)
+                MountArgs=$(echo "$line"|cut -d ' ' -f2-)
                 ;;
             Echo:)
-                echo "$(echo $line|cut -d ' ' -f2-)"
+                echo "$(echo "$line"|cut -d ' ' -f2-)"
```

Comment:
echo "$(echo "$line"|cut -d ' ' -f2-)"
                         ^-- SC2005: Useless echo? Instead of 'echo $(cmd)', just use 'cmd'.